### PR TITLE
Added playground for Grade component

### PR DESCRIPTION
**Pull request by komal-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

-Configure the library and set up the Playground Tab in the layout

-Configure the Playground tab

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2024-07-19 at 3 38 45 PM
Screenshot 2024-07-19 at 3 39 20 PM

#### To Test:

-yarn prettier
-yarn docs-start

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        initialValue: 'A+',
+        description: 'The text shown in the circle',
+        required: true,
+        category: 'Required Props',
+    },
+    // Optional Props
+    {
+        id: 'fontColor',
+        type: 'color',
+        initialValue: 'white',
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx:30]: we need to verify if initialValue should have hex color

File: docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        typeLabel: 'string',
+        description: 'The color of the text label',
+        required: false,
+        category: 'Optional Props',
+    },
+    {
+        id: 'backgroundColor',
+        type: 'color',
+        initialValue: 'green',
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx:39]: we need to verify if initialValue should have hex color

File: docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx (hunk 1 above)
Comment (ArshdeepSingh3-eaton) [reply docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx:30]: it should not

Comment (komal-eaton) [reply docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx]: Added defaultValue.

File: docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    },
+    // Optional Props
+    {
+        id: 'fontColor',
+        type: 'color',
+        typeLabel: 'string',
+        description: 'The color of the text label',
+        required: false,
+        category: 'Optional Props',
+        defaultValue: 'inherit',
```
Comment (ektaghag-eaton) [inline docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx:34]: Can we add defaultValue for fontColor prop as per A+ grade i.e. `#FFFFFF` ?

File: docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        allowMuiColors: true,
+    },
+    {
+        id: 'backgroundColor',
+        type: 'color',
+        typeLabel: 'string',
+        description: 'The background color of the circle',
+        required: false,
+        category: 'Optional Props',
+        defaultValue: 'inherit',
```
Comment (ektaghag-eaton) [inline docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx:44]: Can we add defaultValue for backgroundColor prop as per A+ grade i.e. `#198900`?

File: docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        type: 'number',
+        typeLabel: `number`,
+        initialValue: 40,
+        description: 'The size of the icon',
+        required: false,
+        minValue: 10,
+        maxValue: 60,
+        valueStep: 4,
+        category: 'Optional Props',
+        defaultValue: 'inherit',
```
Comment (ektaghag-eaton) [inline docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx:58]: It should be `40` as per API docs

**Comment by ektaghag-eaton** [comment]
Othe changes looks nice  👍

File: docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx (hunk 3 above)
Comment (ArshdeepSingh3-eaton) [reply docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx:34]: @ektaghag-eaton  This a component that takes custom Label and is not a subcomponent A+ grade

Comment (komal-eaton) [reply docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx:34]: Replaced A+ with H to avoid confusion

**Review by ArshdeepSingh3-eaton — APPROVED** [review]

**Comment by ektaghag-eaton** [comment]
fix conflicts and merge PR.

**Pull request merged**